code/game/machinery/doors/door.dm

```diff
@@ -98,7 +98,14 @@
 			return TRUE
 	return FALSE
 
-/// Triggered when attempting to remove an airlock seal
+/**
+  * Called when attempting to remove the seal from an airlock
+  *
+  * Here because we need to call it and return if there was a seal so we don't try to open the door
+  * or try it's safety lock while it's sealed
```

Comment:
```suggestion
  * or try its safety lock while it's sealed
```

Reply:
Yes, I am being that pedantic! Muahahaha.

He, his. Her, hers. It, its.

Reply:
fuck, how did I not notice this